docs/user-guide/write-data.md

```diff
@@ -2,6 +2,8 @@
 
 Learn how to write data to GreptimeDB. [Connection](./clients.md#connect) is needed before the following operations.
 
+GreptimeDB provides schemaless writing that automatically creates storage structures for your data, so that you don't need to create tables in advance. The table and columns will be created automatically when writing data with protocol [gRPC](#grpc), [InfluxDB](#influxdb-line-protocol), [OpenTSDB](#opentsdb-line-protocol) and [Prometheus remote write](#prometheus). When necessary, GreptimeDB automatically adds the required columns to ensure that the data written by the user is stored correctly.
```

Comment:
1. `storage structures` -> `schemas`
2. `The table and columns will be created` -> `The table and new columns will be created and added`.